src/Http/Http/src/Microsoft.AspNetCore.Http.csproj

```diff
@@ -25,6 +26,7 @@
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore.Connections.Abstractions" />
     <Reference Include="Microsoft.AspNetCore.Http.Abstractions" />
+    <Reference Include="Microsoft.AspNetCore.Routing" />
```

Comment:
WithMetadata extension method on IEndpoinConvemtionBuilder exists on this assembly.

Reply:
That's easy to avoid.
https://github.com/dotnet/aspnetcore/blob/e67e77d85e74c94fd72a942b3b8e77c74c228cd6/src/Http/Routing/src/Builder/RoutingEndpointConventionBuilderExtensions.cs#L81-L92
`builder.Add(b => b.Metadata.Add(attribute));`